extensions/amp-analytics/0.1/vendors.js

```diff
@@ -349,7 +349,41 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'image': true,
     },
   },
-
+	
+'catchpoint': {
```

Comment:
Please fix indentation. Otherwise the lint check won't let us merge this PR. 

Could you please refer to https://travis-ci.org/ampproject/amphtml/jobs/400425441, or run `gulp lint --files=extensions/amp-analytics/0.1/vendor.js` locally to fix the lint errors. Thanks.